spaghetti/network.py

```diff
@@ -1,12 +1,19 @@
 from collections import defaultdict, OrderedDict
 import os
-import cPickle
+try:
+    try:
+        import cPickle as pickle
+    except:
+        import _pickle as pickle
+except:
```

Comment:
I don't think you need this last except?

Reply:
Would it not be comprehensive to have `import pickle` in case the user has neither `cPickle` or `_pickle` installed?

Reply:
Actually, should we get rid of `import cPickle` since we are only supporting python 3 now?

Reply:
@ljwolf How do you think about the above two comment?